app/controllers/consultations_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ConsultationsController < ApplicationController
```

Comment:
Not being used rn, but we'll
May suggestion is to create a patient show, and allow modifying only the relevant information ie;  treatments